front/app/modules/commercial/content_builder/admin/components/CraftComponents/Text/index.tsx

```diff
@@ -8,16 +9,22 @@ import QuillEditor from 'components/UI/QuillEditor';
 import { useNode } from '@craftjs/core';
 import { Box } from '@citizenlab/cl2-component-library';
 
+const StyledTextBox = styled(Box)`
```

Comment:
This is indeed the correct approach but now, it will only apply to the Text component. 

So instead - we should add this to the RenderNode - it's a component that wraps around all craft components. Also - the cursor can always be move - no need to have it as a hover effect, I think.

Reply:
Good points. I will adapt the code.

Reply:
Done. Now the `move` cursor is shown on `RenderNode`.